google/ggrc-core
# GGRC-4290 User role in Assessment is not displayed in UI after importing a comment

On hold until next dependencies are merged:
- [ ] [GGRC-5266](https://gojira.corp.google.com/browse/GGRC-5266)

# Issue description

User role for a comment in Assessment is not displayed in UI after importing a comment. But it is displayed if changed over UI.

# Steps to test the changes

1. Export an Assessment
2. Modify CSV: Ensure current user has any role inside the Assessment. Add a comment.
3. Import the file.
Expected result: the comment is added to the Assessment with the correct role near the comment author. See screenshot-2.png in [GGRC-3505](https://gojira.corp.google.com/browse/GGRC-3505).

# Solution description

The solution was discussed in the [GGRC-3505](https://gojira.corp.google.com/browse/GGRC-3505) ticket comments.
Implement 3rd sequence of importing data:
- attributes;
- mappings;
- comments.

# Sanity checklist




# PR Review checklist
